lib/changeling/support/search.rb

```diff
@@ -11,9 +11,9 @@ def self.find_by(args)
         size = args[:size] || 10
 
         @class = Changeling::Models::Logling
-        @class.tire.index.refresh
+        @class.__elasticsearch__.refresh_index!(index: @class.index_name)
 
-        results = @class.search :per_page => size do
+        results = @class.search nil, {index: @class.index_name} do
```

Comment:
@hahuang65, this is the part that I have a problem figuring out on how to convert it to ES-Rails